### PR TITLE
add grf extension to recent R easyconfigs

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in total)
node3302.joltik.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6242 CPU @ 2.80GHz (cascadelake), Python 3.6.8
See https://gist.github.com/69744a31ef83bf83ac2b83683bcac08a for a full test report.

File: easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb [hunk 1]
```diff
@@ -2939,11 +2939,11 @@ exts_list = [
     ('cSEM', '0.4.0', {
         'checksums': ['7753ac7db9d2c0392e51dd31ec8638e1a7fcbb2546dd9103f5ecc03dd51836c1'],
     }),
-    ('cubelyr', '1.0.1', {
```
Comment (branfosj) [inline easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb:2942]: Please restore this extension.

File: easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb [hunk 2]
```diff
@@ -2939,11 +2939,11 @@ exts_list = [
...
-        'checksums': ['740a34100592b2c6b7bc89a31bddccf4c8fd95720caf68f530104f17aada77bc'],
+    ('DiceKriging', '1.6.0', {
+        'checksums': ['ab5d1332809f2bb16d156ed234b102eb9fbd6de792e4291f9f6ea4652215cb49'],
     }),
-    ('broom.mixed', '0.2.6', {
```
Comment (branfosj) [inline easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb:2945]: And this one.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/r/R/R-4.1.0-foss-2021a.eb:2942]: Muh, no idea how that happened... Will fix.

**Comment by boegel** [comment]
@boegelbot please test @ generoso
EB_ARGS="R-4.0.3-foss-2020b.eb R-4.0.4-foss-2020b.eb R-4.0.5-foss-2020b.eb R-4.1.0-foss-2021a.eb --skip"

**Review by branfosj — APPROVED** [review]

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
bear-pg0206u05a.bear.cluster - Linux RHEL 8.3, x86_64, Intel(R) Xeon(R) Gold 6248 CPU @ 2.50GHz (cascadelake), Python 3.6.8
See https://gist.github.com/d8bc06dd734f1abb4cb164db48622ecc for a full test report.

**Comment by branfosj** [comment]
Going in, thanks @boegel!

**Pull request merged**